src/main/java/raisetech/student/management/controller/StudentController.java

```diff
@@ -5,24 +5,30 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
+import raisetech.student.management.controller.converter.StudentConverter;
 import raisetech.student.management.data.Student;
 import raisetech.student.management.data.StudentsCourses;
+import raisetech.student.management.domain.StudentDetail;
 import raisetech.student.management.service.StudentService;
 
 @RestController
 public class StudentController {
 
   private StudentService service;
+  private StudentConverter converter;
 
   @Autowired
-  public StudentController(StudentService service) {
+  public StudentController(StudentService service, StudentConverter converter) {
     this.service = service;
+    this.converter = converter;
   }
 
   //	生徒情報を表示する
   @GetMapping("/studentList")
-  public List<Student> getStudentList(@RequestParam String age) {
-    return service.searchStudentList(age);
+  public List<StudentDetail> getStudentList(@RequestParam String age,String course) {
```

Comment:
細かいですけども。
```suggestion
  public List<StudentDetail> getStudentList(@RequestParam String age, String course) {
```
保存時自動でのコードフォーマットを設定しておくとよいです。